.github/dependabot.yml

```diff
@@ -50,7 +50,7 @@ updates:
   - package-ecosystem: "github-actions"
     directory: "/"
     schedule:
-      interval: weekly
+      interval: "weekly"
```

Comment:
you mention making things consistent, but missed updating the settings below.

Reply:
yeah, added for rest of the labels too.